phylanx/execution_tree/primitives/inverse_operation.hpp

```diff
@@ -1,38 +1,38 @@
-//   Copyright (c) 2017 Hartmut Kaiser
-//
-//   Distributed under the Boost Software License, Version 1.0. (See accompanying
-//   file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-
-#if !defined(PHYLANX_PRIMITIVES_INVERSE_OPERATION_OCT_09_2017_0154PM)
-#define PHYLANX_PRIMITIVES_INVERSE_OPERATION_OCT_09_2017_0154PM
-
-#include <phylanx/config.hpp>
-#include <phylanx/ast/node.hpp>
-#include <phylanx/execution_tree/primitives/base_primitive.hpp>
-#include <phylanx/ir/node_data.hpp>
-
-#include <hpx/include/components.hpp>
-
-#include <vector>
-
-namespace phylanx { namespace execution_tree { namespace primitives
-{
-    class HPX_COMPONENT_EXPORT inverse_operation
-      : public base_primitive
-      , public hpx::components::component_base<inverse_operation>
-    {
-    public:
-        static std::vector<match_pattern_type> const match_data;
-
-        inverse_operation() = default;
-
-        inverse_operation(std::vector<primitive_argument_type>&& operands);
-
-        hpx::future<primitive_result_type> eval() const override;
-
-    private:
-        std::vector<primitive_argument_type> operands_;
-    };
-}}}
-
-#endif
+////   Copyright (c) 2017 Hartmut Kaiser
+////
+////   Distributed under the Boost Software License, Version 1.0. (See accompanying
+////   file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+//#if !defined(PHYLANX_PRIMITIVES_INVERSE_OPERATION_OCT_09_2017_0154PM)
+//#define PHYLANX_PRIMITIVES_INVERSE_OPERATION_OCT_09_2017_0154PM
+//
+//#include <phylanx/config.hpp>
+//#include <phylanx/ast/node.hpp>
+//#include <phylanx/execution_tree/primitives/base_primitive.hpp>
+//#include <phylanx/ir/node_data.hpp>
+//
+//#include <hpx/include/components.hpp>
+//
+//#include <vector>
+//
+//namespace phylanx { namespace execution_tree { namespace primitives
+//{
+//    class HPX_COMPONENT_EXPORT inverse_operation
+//      : public base_primitive
+//      , public hpx::components::component_base<inverse_operation>
+//    {
+//    public:
+//        static std::vector<match_pattern_type> const match_data;
+//
+//        inverse_operation() = default;
+//
+//        inverse_operation(std::vector<primitive_argument_type>&& operands);
+//
+//        hpx::future<primitive_result_type> eval() const override;
+//
+//    private:
+//        std::vector<primitive_argument_type> operands_;
+//    };
+//}}}
+//
+//#endif
```

Comment:
This is still commented out, was that intentional?

Reply:
Yes, matrix inversion and determinant need BLAS. I disabled this test.